packages/api/src/auth/authDecoders/goTrue.ts

```diff
@@ -0,0 +1,54 @@
+import type {
```

Comment:
I think it's OK for netlify and gotrue to share decoders.

Reply:
Yes, those were bad smells.